config/wpe_config.toml

```diff
@@ -86,3 +86,6 @@ connection_retry = 3
 [WorkerConfig]
 # Id of worker in plain text
 worker_id = "kme-worker-1"
+# Supported workloads for this worker. This can be overridden by passing
+# values in command line for enclave manager using --workloads option.
+workloads = "echo-result,heart-disease-eval,inside-out-eval,simple-wallet"
```

Comment:
We have to update adding new workload tutorial document also otherwise tests will.